README.md

```diff
@@ -0,0 +1,271 @@
+# *hgvs* - manipulate biological sequence variants according to Human Genome Variation Society recommendations
+
+**Important:** biocommons packages require Python 3.6+.
+[More](https://groups.google.com/forum/#!topic/hgvs-discuss/iLUzjzoD-28)
+
+The *hgvs* package provides a Python library to parse, format, validate,
+normalize, and map sequence variants according to [Variation
+Nomenclature](http://varnomen.hgvs.org/) (aka Human Genome Variation
+Society) recommendations.
+
+Specifically, the hgvs package focuses on the subset of the HGVS
+recommendations that precisely describe sequence-level variation
+relevant to the application of high-throughput sequencing to clinical
+diagnostics. The package does not attempt to cover the full scope of
+HGVS recommendations. Please refer to
+[issues](https://github.com/biocommons/hgvs/issues) for limitations.
+
+### **Information**
+
+[![rtd](https://img.shields.io/badge/docs-readthedocs-green.svg)](http://hgvs.readthedocs.io/) [![changelog](https://img.shields.io/badge/docs-changelog-green.svg)](https://hgvs.readthedocs.io/en/stable/changelog/)  [![getting_help](https://img.shields.io/badge/!-help%20me-red.svg)](https://hgvs.readthedocs.io/en/stable/getting_help.html)  [![GitHub license](https://img.shields.io/github/license/biocommons/hgvs.svg)](https://github.com/biocommons/hgvs/blob/main/LICENSE)  [![binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/biocommons/hgvs/main?filepath=examples)
+
+### **Latest Release**
+
+[![GitHub tag](https://img.shields.io/github/tag/biocommons/hgvs.svg)](https://github.com/biocommons/hgvs) [![pypi_rel](https://img.shields.io/pypi/v/hgvs.svg)](https://pypi.org/project/hgvs/)
+
+### **Development**
+
+[![coveralls](https://img.shields.io/coveralls/github/biocommons/hgvs.svg)](https://coveralls.io/github/biocommons/hgvs) [![issues](https://img.shields.io/github/issues-raw/biocommons/hgvs.svg)](https://github.com/biocommons/hgvs/issues)
+[![GitHub Open Pull Requests](https://img.shields.io/github/issues-pr/biocommons/hgvs.svg)](https://github.com/biocommons/hgvs/pull/) [![GitHub license](https://img.shields.io/github/contributors/biocommons/hgvs.svg)](https://github.com/biocommons/hgvs/graphs/contributors/) [![GitHub stars](https://img.shields.io/github/stars/biocommons/hgvs.svg?style=social&label=Stars)](https://github.com/biocommons/hgvs/stargazers) [![GitHub forks](https://img.shields.io/github/forks/biocommons/hgvs.svg?style=social&label=Forks)](https://github.com/biocommons/hgvs/network)
+
+## **Features**
+
+-   Parsing is based on formal grammar.
+-   An easy-to-use object model that represents most variant types
+    (SNVs, indels, dups, inverstions, etc) and concepts (intronic
+    offsets, uncertain positions, intervals)
+-   A variant normalizer that rewrites variants in canoncial forms and
```

Comment:
```suggestion
-   A variant normalizer that rewrites variants in canonical forms and
```

Reply:
LOL didn't realize how many typos there were in this! I was so focused on making sure all of the formatting was fine after using Pandoc to convert it to md. Good catches!